dev/com.ibm.ws.security.oidc.client_fat.2/fat/src/com/ibm/ws/security/openidconnect/client/fat/IBM/OidcClientWasReqURLTests.java

```diff
@@ -789,46 +789,46 @@ public void testWasReqURLOidc_cookie(String updatedHost, String rpServerConfig,
-                                                    MessageConstants.CWOAU0073E_FRONT_END_ERROR);
-                expectations = validationTools.addMessageExpectation(testRPServer, expectations, Constants.LOGIN_USER, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS,
-                                                                     "Server log did not contain an error message about the missing WASReqURLOidc cookie.",
-                                                                     MessageConstants.CWWKS1532E_MALFORMED_URL_IN_COOKIE + ".*" + updatedHost + ".*");
+            expectations = vData.addExpectation(expectations, Constants.LOGIN_USER, Constants.RESPONSE_FULL, Constants.STRING_CONTAINS,
+                    "Did not receive error message " + MessageConstants.CWOAU0073E_FRONT_END_ERROR + " in the response", null,
+                    MessageConstants.CWOAU0073E_FRONT_END_ERROR);
+            expectations = validationTools.addMessageExpectation(testRPServer, expectations, Constants.LOGIN_USER, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS,
+                    "Server log did not contain an error message about the missing WASReqURLOidc cookie.",
+                    MessageConstants.CWWKS1532E_MALFORMED_URL_IN_COOKIE + ".*" + updatedHost + ".*");
 
             break;
         case EXCEPTION:
             expectations = vData.addSuccessStatusCodesForActions(expectations, Constants.LOGIN_USER, Constants.GOOD_OIDC_LOGIN_ACTIONS_SKIP_CONSENT);
-            // timeout/time out is no longer appearing in the connection refused message - just looking for Connection refused now
+            // timeout/time out is no longer appearing in the connection refused message - just looking for Connection refused now, Java 18 has yet another variation
             //            // have been getting an exception with "timed out" in it, now we have a jdk that is returning "timeout"
             //            // the exception checking code can't handle matches (to use ".*time.*out.*") so, we'll use 2 checks
             //                expectations = vData.addExpectation(expectations, Constants.LOGIN_USER, Constants.EXCEPTION_MESSAGE, Constants.STRING_CONTAINS, "Did NOT get expected time out exception",
             //                                                    null, "time");
             //                 expectations = vData.addExpectation(expectations, Constants.LOGIN_USER, Constants.EXCEPTION_MESSAGE, Constants.STRING_CONTAINS, "Did NOT get expected time out exception",
             //                                                    null, "out");
             expectations = vData.addExpectation(expectations, Constants.LOGIN_USER, Constants.EXCEPTION_MESSAGE, Constants.STRING_CONTAINS, "Did NOT get expected \"Connection refused\" exception",
-                    null, "Connection refused");
+                    "java.net.ConnectException", "Connection refused");
```

Comment:
If the `Constants.EXCEPTION_MESSAGE` type supports the `Constants.STRING_MATCHES` search method, this could be updated to be a regex with any number of options to look for.